src/go/install.sh

```diff
@@ -219,8 +219,18 @@ if [ "${INSTALL_GO_TOOLS}" = "true" ]; then
 
     # Move Go tools into path and clean up
     mv /tmp/gotools/bin/* ${TARGET_GOPATH}/bin/
-
     rm -rf /tmp/gotools
+
+    # Install golangci-lint from precompiled binares
+    if [ "$GOLANGCILINT_VERSION" = "latest" ] || [ "$GOLANGCILINT_VERSION" = "" ]; then
+        echo "Installing golangci-lint latest..."
+        curl -fsSL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | \
+            sh -s -- -b "/usr/local/bin"
```

Comment:
I suppose this should point to the `GO` path? 

```suggestion
            sh -s -- -b "/usr/local/go/bin"
```

Reply:
I would say no, because `/usr/local/go/bin` is for binaries that were installed by `go install`. Technically this is not installed through that, so I think keeping it oustide of the gopath makes most sense.

Reply:
Your point makes sense to me, however, I was trying to be consistent with the doc pointed out in the PR description. It seems to install in `GO` path which seems appropriate to me - having all `go` utils in one place. 

https://golangci-lint.run/usage/install/#linux-and-windows

Reply:
Ok. In that case, it should probably be `${TARGET_GOPATH}/bin`. Updated